python/triton/runtime/backends/cuda.c

```diff
@@ -17,9 +17,7 @@ static inline void gpuAssert(CUresult code, const char *file, int line) {
 
 #define CUDA_CHECK(ans)                                                        \
   {                                                                            \
-    gpuAssert((ans), __FILE__, __LINE__);                                      \
-    if (PyErr_Occurred())                                                      \
-      return NULL;                                                             \
+    { gpuAssert((ans), __FILE__, __LINE__); }                                  \
```

Comment:
I audited the use in this file and I couldn't tell any reason for checking `PyErr_Occurred`. This also makes this macro line behave the same as other `#define CUDA_CHECK` inside triton